mission/functions/systems/actions/fn_action_gather_intel.sqf

```diff
@@ -1,19 +1,66 @@
+private _object = player;
+private _title = "Gather Enemy Intel";
+private _iconIdle = "custom\holdactions\holdAction_documents_ca.paa";
+private _iconProgress = "custom\holdactions\holdAction_documents_ca.paa";
+
+
+// this is a hacky workaround following similar logic to this github gist
+// https://gist.github.com/CalebSerafin/f4a16f2f564a438882ca6f32107bec24
+// which tries to solve this issue
+// https://feedback.bistudio.com/T155638
+//
+// however, instead of relying on undocumented functionality
+// (which could disappear in the future!) i've changed it so that the
+// holdAction UI is always redrawn every 5-ish seconds.
```

Comment:
tl;dr version -- we hack the holdaction to not be shown for one frame, but still continue progression of the hold action.

Reply:
stopping the UI from being shown for one frame apparently doesn't block progression so :shrug: